src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,16 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
-
+    public static void main(String[] args) {
+        final int radius = 15;
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[5];
+        for (int i = 0; i <= 4; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        figures[5] = new Circle("Black", radius);
```

Comment:
Our default `Circle` should be white color. Also, please take a look at task description

> The other half of figures should have the same, default parameters

Now, you will simply get AIOBE